UnityProject/Assets/Scripts/Items/Attributes.cs

```diff
@@ -99,13 +99,13 @@ private void SyncArticleDescription(string newDescription)
 	public void OnHoverStart()
 	{
 		UIManager.SetToolTip =
-			initialName.First().ToString().ToUpper() + initialName.Substring(1) + 
-			(String.IsNullOrEmpty(articleDescription) ? "" : $" ({articleDescription})");
+			initialName.First().ToString().ToUpper() + initialName.Substring(1) +
```

Comment:
This causes a "sequence is empty" exception in the logs when mousing over some things, I think when they have empty string for their name? There's something in the shuttle that I can mouse over to cause this.

Reply:
nvm I'll fix this myself since I missed it when I reviewed another PR